### DISCUSSION TITLE
INAV with Props Out (Reversed) ? And More Stuffs...

### DISCUSSION THREAD

**Discussion by EricMangus** [iNavFlight/inav#6449] [Q&A] [answered]
Hi, I've been looking everywhere on the web and YouTube for an answer to this question, but I can't find it :: **Can INAV work with a quadcopter setup with reversed propellers (Props Out) configuration ?**

For Context :
I'm building/modifying an iFlight Chimera 4LR BNF to run INAV 2.6. I replaced the iFlight SucceX flight controller with a Matek F722-mini SE and I also replaced the GPS Unit with a Matek M8Q-5883. The Original iFlight Chimera 4 was setup with BetaFlight with a Props Out/Reverse Propeller configuration. I kept this motor rotation configuration to maintain design consistency... 

I Flashed the Matek F722 Mini SE flight controller with MATEKF722mini v,2.6 and setup the motor output to DSHOT 300. Once everything was setup and configured - when I arm and begin to raise the throttle to reach a hover, the drone does a fast Yaw turn to the Right at which point I disarm to avoid making the situation worse. I double-checked that I did not have any of the propellers on backwards etc. I did a previous successful INAV 2.5 build on an eACHINE Tyro 129 and I used Multishot as the motor output. So I figured this may be worth a try. I changed the motor output to Multishot - but when I arm and raise the throttle, nothing happens // Super Super Strange...

Now I'm starting to second guess myself a little bit. Without touching any wires or hardware whatsover, I flashed the Matek FC back to BetaFlight and left the motor output to DSHOT 300. _I arm and everything is working great ?!_ 
The only thing I'm thinking that is different is I specified Reverse Props in BetaFlight. The Tyro 129 with iNav :: I used the traditional props in (normal motor rotation configuration). So the only logical conclusion seems to be that INAV probably does not allow Props Out ???

I really want this drone to fly with INAV with preferably with the latest firmware too... Any Help from the Pros and/or Masters of Disasters would be Much Appreciated 👍. Thanks and Happy Holidays.

**Comment by EricMangus** [comment, marked as answer]
I Think I Found the Solution: Watching **Paweł Spychalski's** "_Betaflight to INAV in 22 minutes_" Mr. Spychalski mentions that you can use **motor_direction_inverted = ON** in the Command Line Interface. I'll try again and see what happens now... Thank Goodness for your videos Paweł : )

**Reply by EricMangus** [reply]
Confirmed that **motor_direction_inverted = ON** and using DSHOT300 worked. Now the problem that I'm having is NAV POSHOLD...

I activate the channel on my radio for NAV POSHOLD and the Chimera still moves around in Angle Mode as if it's not doing anything (gps position holding) - it just drifts around ?

I saw on the Mateksys website for the M8Q-5883 it says to keep the GPS 10 cm (3.937 / 4 inches) away from anything metallic. On the Tyro 129 I had space to install a GPS Antenna Mast. I'm using the original TPU printed Immortal-T holder plus GPS holder that came provided with the iFlight Chimera 4 LR... I am able to ARM the quad which would mean that I have enough Satellites for Navigation ? This quad is a 4-inch long range - I don't think I have the space to mount an Antenna Mast. Have people tested NAV POSHOLD on INAV v.2.6 ? Any tips and advise would be greatly appreciated. **THANKS**

**Comment by DzikuVx** [comment]
yeap, that's the correct answer

**Reply by Gaz-B** [reply]
Should the image in the configurator now show after the command as it still shows props in ?.

**Reply by DzikuVx** [reply]
No, image is not updated

**Reply by palgy** [reply]
can we have an updated image ?

**Reply by Gaz-B** [reply]
It would be nice to have a handshake to confirm the command has worked. As is the image is instructing the user to fit them incorrectly once reversed. I armed with no props on and moved the yaw axis to confirm which sped and which slowed to be sure.
